source/publishing_data/11_saving_and_publishing_a_dataset/dataset_actions_status.rst

```diff
@@ -0,0 +1,69 @@
+Dataset actions and statuses
+============================
+
+Actions
+~~~~~~~
+
+**Actions related to the life cycle of a dataset**
+
+.. list-table::
+  :header-rows: 1
+
+  * * Action
+    * Button
+    * Description
+  * * Save
+    * .. image:: images/button_save.png
+    * Saves all new modifications, processings and configurations operated on the dataset.
+  * * Publish
+    * .. image:: images/button_publish.png
+    * Makes the dataset's modifications go live on the portal. The duration of the publishing varies depending on the dataset, and the nature of the modifications.
```

Comment:
We could explain that in some situations (schema changes) all data will be erased during the processing.
It could lead to date lost in case of realtime datasets and some data sources.